tests/test_constants.py

```diff
@@ -0,0 +1,66 @@
+"""Test constants."""
```

Comment:
This is some very familiar looking code. Can't wait to get this using dmlutils. Exciting days ahead.